iron-ajax.html

```diff
@@ -357,7 +350,7 @@
       request.completes.then(
         this._boundHandleResponse
       ).catch(
-        this._boundHandleError
+        this.handleError.bind(this, request)
```

Comment:
Should the other `_boundXXX` methods get the same treatment? (i.e. get bound in here, not in a separate method?)